modules/gapi/include/opencv2/gapi/streaming/source.hpp

```diff
@@ -55,6 +55,9 @@ IStreamSource::Ptr inline make_src(Args&&... args)
     return src_ptr->ptr();
 }
 
+// NB: Python wrapper can't parse class::Ptr, so need to use using
+using IStreamSourcePtr = cv::gapi::wip::IStreamSource::Ptr;
```

Comment:
cv::Ptr<> can be parsed

Reply:
Yes, but class::Ptr can't be parsed

Reply:
Do we need to define `IStreamSourcePtr` anyway?

Reply:
Yes, because `make_capture_src` return `IStreamSource::Ptr` which can't be parsed

Reply:
What if we will use
```cpp
GAPI_EXPORTS_W Ptr<IStreamSource> inline make_capture_src(const String& path)
```
?

Reply:
IStreamSource::Ptr is `shared_ptr` not `cv::Ptr`

Reply:
`cv::Ptr` is `std::shared_ptr` on master branch: https://docs.opencv.org/master/dc/d84/group__core__basic.html#ga6395ca871a678020c4a31fadf7e8cc63

Reply:
GRunArg is `variant<..., std::shared_ptr<IStreamSource>> which can't take `cv::Ptr` :(

Reply:
Fixed